### ISSUE TITLE
Font color in dark mode

### ISSUE THREAD

**Issue by Jerfain** [Shirajuki/anki-redesign#27]
<img width="711" alt="图片" src="https://user-images.githubusercontent.com/69144012/177176039-af850b1f-6400-4fae-a863-878566aa1d8a.png">
^ As shown above, in the previous version of redesign the font color was white in dark mode.
<img width="1127" alt="图片" src="https://user-images.githubusercontent.com/69144012/177176063-61d66157-b0e2-4726-a22a-6a2924e49b13.png">
^ However, in the latest version of redesign the font color becomes black in the dark mode, thus making it hard to see, please tell me which variable I should change in the settings.

**Comment by Jerfain** [comment]
I suddenly found a new discovery, when I switch the theme to Anki and then switch back to Catppuccin, it is back to normal... Then I suspected that it was a problem in the automatic switching but I tried and it didn't reproduce the bug, well, it's probably just a random glitch.

**Comment by Shirajuki** [comment]
Hello again @Jerfain!
I feel bad that there exists lots of bugs still 🙃... to tell you the truth this is a known bug that I discovered when working on the last update and couldn't seem to get it fixed no matter what I tried. This is a problem that came with the release of QT6 and randomly happens every 4th time or so when you swap the theme or change from dark/light. I've made it happen less frequently than before, and felt that it was "good enough" to be pushed out in the latest release since I thought one rarely swaps themes, but looks like it came to bite me back as the bug is actually still not completely fixed. 🙃

I am currently looking for a fix for this, and hopefully the next update release of Anki-redesign includes a fix for this issue as well. Again, thank you very much for contributing to Anki-redesign!
